### ISSUE TITLE
Drag and drop media timeout issue

### ISSUE THREAD

**Issue by alfonso-jimenez-evelyn** [umbraco/Umbraco-CMS#14864]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

12.0.1

### Bug summary

When I use the "drag and drop"  to upload a list of files, it is loading to 100% each file but no further. After a time, a timeout error appears.
![image](https://github.com/umbraco/Umbraco-CMS/assets/88671313/6cf782cd-0d7c-48bb-bc88-7615c29f0628)
![image](https://github.com/umbraco/Umbraco-CMS/assets/88671313/548104dc-60a9-4879-96b1-e6cff106b940)
The issue is fixed when I restart my web app, but after few days or when I uploaded a big list of files appears again. It's like an error blocked the PostAddFile method.

### Specifics

- Each file size uploaded is not too big (less than 2mb)
- The following app settings had been set : 
```
  "Umbraco": {
    "CMS": {
      "Runtime": {
        "MaxRequestLength": 5000
      },
      "Hosting": {
        "Debug": true,
        "LocalTempStorageLocation": "EnvironmentTemp"
      },
      "RuntimeMinification": {
        "UseInMemoryCache": true,
        "CacheBuster": "AppDomain"
      }
    },
```

### Steps to reproduce

1. Drag and drop a list of files.
2. Wait for response (Timeout error)
3. If one of the files returns error, then is not possible upload more files.

### Expected result / actual result

The drag and drop option it should works without restarting the web app several times.
I'll glad to read any help related with, thanks.

**Comment by andr317c** [comment]
Hey!

Thanks for reporting this issue. I was not able to reproduce it, I tried on versions 12.0.1 and 12.2.0. How many files are you trying to upload at the same time? 😄

**Comment by alfonso-jimenez-evelyn** [comment]
Thanks for repplying me, it's already fixed this issue in v12.2.0
